openhands/core/config/app_config.py

```diff
@@ -49,7 +49,7 @@ class AppConfig:
     default_agent: str = OH_DEFAULT_AGENT
     sandbox: SandboxConfig = field(default_factory=SandboxConfig)
     security: SecurityConfig = field(default_factory=SecurityConfig)
-    runtime: str = 'eventstream'
+    runtime: str = 'docker'
```

Comment:
Can we do a global replace in evaluation folder as well?

Reply:
done!